core/sagas/inputs.js

```diff
@@ -41,7 +41,16 @@ function flattenInputs(response) {
 function* fetchInputs(action) {
   try {
     const { filter, selectedInputPage, pageSize } = action.payload;
-    const filterValue = `*${filter.value}*`.replace("**", "*");
+    const wildcardsUsed = filter.value.includes("*");
+    const regex = / +|, +/g;
+    let filterValue = filter.value.replace(regex, ",");
+    if (filterValue.charAt(0) === ",") {
+      filterValue = filterValue.slice(1);
+    }
+    if (filterValue.charAt(filterValue.length - 1) === ",") {
+      filterValue = filterValue.slice(0, filterValue.length - 1);
+    }
```

Comment:
I would replace this with a regex to remove multiple leading and following commas. A user could enter ` ,dog` which would be replaced to `,,dog` and then stripped to `,dog` and then replaced to  `**,*dog*`. Maybe the input box should be restricted here to alert a user they can't enter leading commas. The regex I would use to strip the filter is 

`stripRegex = /(^,+)|(,+$)/g`

Reply:
👍 This suggestion is amazing! Thanks for your help.

Reply:
I committed your suggestion, tested it, and squashed all commits.